src/gui/widgets/EffectLabelButton.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * EffectLabelButton.cpp - implementation of class trackLabelButton, a label
+ *                          which is renamable by double-clicking it
+ *
+ * Copyright (c) 2004-2008 Tobias Doerffel <tobydox/at/users.sourceforge.net>
+ * 
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+
+#include "EffectLabelButton.h"
+
+#include <QMouseEvent>
+#include <QMdiSubWindow>
+
+#include "ConfigManager.h"
+#include "embed.h"
+#include "EffectView.h"
+#include "Effect.h"
+
+namespace lmms::gui
+{
+
+EffectLabelButton::EffectLabelButton(EffectView* _tv, QWidget* _parent) :
+	QPushButton(_parent),
+	m_effectView(_tv),
+	m_iconName()
+{
+	setAttribute(Qt::WA_OpaquePaintEvent, true);
+	setAcceptDrops(false);
+
+	setCursor(QCursor(embed::getIconPixmap("hand"), 3, 3));
+	setStyleSheet("text-align:left;padding:2px;");
```

Comment:
This line should be removed and the definition should be done in the style sheets of the default and classic theme. This way users can override the style.

Did you check the look of the changes with the classic theme by the way?

Reply:
This line is identical to the one in the `TrackLabelButton` class (which I based this class off of), so I think for consistency it makes sense to keep it this way unless we want to change both of them.

Reply:
Added classic theme screenshots to the top message. As an aside, it looks like the recent mixer strip PR was not tested in the classic theme since it's completely black...

Reply:
You're right. I have just checked version 1.2 with the classic theme and here the mixer strip is gray:

![7438-MixerStripClassic](https://github.com/user-attachments/assets/41b54ed2-cd4a-41f9-9509-0df3584aaa49)

Might be something for a new issue.

Reply:
@michaelgregorius did you convert this to use the stylesheet in your local branch? I'm asking because the latest update you posted seems to use the stylesheet.

Reply:
@Rossmaxx, in the message below I simply added some style sheet which sets a font size for `EffectLabelButton`s that are below the effects view, i.e.:
```
lmms--gui--EffectRackView lmms--gui--EffectLabelButton {
	font-size: 12px;
}
```

But it has nothing to do with the mixer which this thread seems to be about. Or am I missing something?

Reply:
I have checked with the current master. The currently selected channel is rendered with a black background in the classic theme:

![7438-SelectedChannelIsBlack](https://github.com/user-attachments/assets/d513ddd8-8ecd-4739-ae7b-8c2f253ae889)

Might be something for a new issue if that is not intended.

Reply:
I was supposed to point out the line in question, not the mixer. But since the line is marked in this review, I can't spin up a new one.

Reply:
The `setStyleSheet` call to be more precise

Reply:
I don't really have a branch for the stuff that I have proposed in this comment: https://github.com/LMMS/lmms/pull/7438#issuecomment-2397205890. It's just some local changes that I have tried out without committing them somewhere.

I have noticed though that my original statement that I have made [here](https://github.com/LMMS/lmms/pull/7438#discussion_r1709831864) is likely wrong anyway because the style sheets are applied after construction and therefore they should be able to override any default style sheet settings made in the constructor anyway.

On a side note: I cannot find any calls to `setStyleSheet` in `TrackLabelButton` in the current master (anymore?): https://github.com/LMMS/lmms/blob/master/src/gui/tracks/TrackLabelButton.cpp. So if `EffectLabelButton` is intended to be similar to that class as indicated in [this comment](https://github.com/LMMS/lmms/pull/7438#discussion_r1710644220) then it might need some adjustment. I have also noticed that both classes differ with regards to their base class but that's something for another PR remark.